src/adapters/api-blueprint-adapter.coffee

```diff
@@ -298,9 +297,9 @@ legacyResourcesFrom1AResource = (legacyUrlConverterFn, resource, sourcemap, opti
       legacyResource.model = {}
 
     legacyResource.resourceParameters = resourceParameters or []
-    legacyResource.actionParameters   = actionParameters or []
+    legacyResource.actionParameters   = getParametersOf(action, options) or []
 
-    legacyResource.parameters = actionParameters or resourceParameters or []
+    legacyResource.parameters = legacyResource.resourceParameters.concat(legacyResource.actionParameters)
```

Comment:
Resource parameters are called `resourceParameters` here. `parameters` doesn't mean anything here now.

Current behaviour of `parameters` is as follows:

* APIB AST -> `actionParameters or resourceParameters`
* Refract -> `resourceParameters.concat(actionParameters)`

What you want to change is to the following for both of them:

`parameters = resourceParameters`

Am I understanding right?